app/src/main/res/layout-land/onboarding_slide.xml

```diff
@@ -47,9 +47,9 @@
       android:id="@+id/slide_description_text_view"
       android:layout_width="0dp"
       android:layout_height="wrap_content"
-      android:layout_marginTop="8dp"
-      android:layout_marginEnd="40dp"
       android:layout_marginStart="10dp"
```

Comment:
This is mentioned as `18dp` in mocks and the end margin is `38dp` so I guess as you have made `38dp` to `40dp`,  18dp can be made as `20dp`